### DISCUSSION TITLE
Préoccupation sur le push

### DISCUSSION THREAD

**Discussion by Ornella242** [dehboris/trombi-srs-2021#42] [General]
J'ai une préoccupation au niveau du git push 
Quand je fais le push ça m'envoi un message d'erreur qui dit permission to dehboris/trombi-al-2021.git denied to Ornella

**Comment by Mitou651** [comment]
Moi aussi j’ai le même problème

**Reply by Ornella242** [reply]
J'ai pu résoudre de mon côté c'était le lien qui était incorrect

**Comment by Mitou651** [comment]
Le miens es correct mais ne passe pas

Le mer. 14 juil. 2021 à 20:12, Ornella242 ***@***.***> a
écrit :
